packages/scene-composer/src/components/three-fiber/CameraComponent/index.tsx

```diff
@@ -47,40 +41,21 @@ const CameraComponent: React.FC<ICameraComponentProps> = ({ node, component }: I
     }
   }, [activeCameraName]);
 
-  useEffect(() => {
-    if (cameraHelperRef.current) {
-      cameraHelperRef.current.visible = isEditing() && !isLoadingModel;
+  const cameraHelper = useMemo(() => {
+    let camera: THREE.OrthographicCamera | THREE.PerspectiveCamera;
```

Comment:
I'll remember that going forward. The only benefit rather than treating it as a namespace is to webpack. Since we do this in numerous places we aren't benefiting from the savings. I'll make a backlog item to go through and replace all instances of `import * as THREE from 'three';` with only those objects being used imported and renamed when appropriate.